llmfoundry/models/mpt/modeling_mpt.py

```diff
@@ -397,12 +428,16 @@ def forward(
                                        dim=1)[:, past_position:],
                     min=0,
                 )
+            if self.rope:
+                rotary_emb_w_offset_info = {
+                    'rotary_emb': self.rotary_embedding,
+                    'pos': pos,
+                    'seq_len': S + past_position
+                }
+            if self.learned_pos_emb:
+                pos_emb = self.wpe(pos)
```

Comment:
```suggestion
            x = tok_emb
            if self.learned_pos_emb:
                pos_emb = self.wpe(pos)
                x = x + pos_emb
```

and remove 
```
pos_emb = 0.0
```
above 
and remove 
```
x = tok_emb + pos_emb
```
below